release-doc-in-process/all_vf.TTL

```diff
@@ -30,6 +30,23 @@ vf:AgentRelationshipRole  a  owl:Class ;
         rdfs:subClassOf   rdf:Property ;
         rdfs:comment   "A relationship role defining the kind of association one agent can have with another." .
 
+#vf:Recipe a                owl:Class ;
+#        rdfs:label         "vf:Recipe" ;
+#        rdfs:comment       "Recipes contain all the info required to create or improve or move a resource." .
+
+vf:ResourceSpecification a owl:Class ;
+        rdfs:label         "vf:ResourceSpecification" ;
+        rdfs:comment       "." .
+
+vf:RecipeProcess a          owl:Class ;
+        rdfs:label         "vf:RecipeProcess" ;
+        rdfs:comment       "." .
+
+vf:RecipeFlow a            owl:Class ;
+        rdfs:label         "vf:RecipeFlow" ;
+        rdfs:comment       "The linkage between resource types, process types, and actions that structure a recipe." .
```

Comment:
Thanks, will fix.  But I think that resource specifications could include types of work and services, whether they specify a material resource or not.  All potential economic resources that could be imagined in a recipe must be described by a resource specification I think.  Maybe the resource specification only has a reference to one resource classification, that could work.

Changed to: "The linkage between a process recipe, an action that structures a recipe, and a resource specification."  What do you think?

Reply:
Let's leave it like that for now. I would not use `ResourceSpecification` since we don't reference any resource from *work* flows and this would just make everything more confusing.
Only flows which at event stage will reference an actual resource with *affects* should use `vf:resourceConformsTo` and reference specification of a resource which event will reference using `vf:affects`. I'll create another issue about categorizing ans specifying work and services.

Reply:
I put it back and changed it to be singular.  Also noticed a couple non-definitions, and added those.  Please check them because you may disagree.  I'm trying to clarify based on this discussion.